waiter/project.clj

```diff
@@ -47,6 +48,7 @@
                   :exclusions [org.clojure/clojure]]
                  [fullcontact/full.async "1.0.0"
                   :exclusions [org.clojure/clojure org.clojure/core.async]]
+                 [hiccup/hiccup "1.0.5"]
```

Comment:
In the past we avoided importing `hiccup` in favor of `comb` to render markup. Does `comb` not have the features you need to generate your markup?

Reply:
done